compiler-utils/src/testFixtures/java/com/squareup/anvil/compiler/internal/testing/TestUtils.kt

```diff
@@ -46,6 +47,9 @@ public class AnvilCompilation internal constructor(
     enableExperimentalAnvilApis: Boolean = true,
   ) = apply {
     checkNotCompiled()
+    check(!anvilConfigured) { "Anvil should not be configured twice." }
```

Comment:
Is this related to the @ContributesTo changes or should this be a separate commit/PR? Seems unrelated to me...

Reply:
It was necessary for one specific test that verifies contributions from the old format are picked up. This is just a sanity check.